src/cli/libs/utils.py

```diff
@@ -272,18 +271,38 @@ def deploy_servers(machines, token, ip_addresses):
     if len(machines) < 1:
         return
 
-    def get_machine_tier(tags):
-        tag_names = [tag.name for tag in tags]
-        for tag_name in tag_names:
-            if "Tier" in tag_name:
-                return tag_name
-        return None
+    def get_node_labels(tags):
+        label_strings = []
+        for tag in tags:
+            tag_name = tag.name
+            if tag_name.startswith("LABEL_"):
+                suffix = tag_name.split("_", 1)[1]
+                label_string = "cnaps.io/" + suffix.replace("_", "=")
```

Comment:
I think this may be clearer?
```suggestion
                _, label_name, label_value = tag_name.split("_", 3)
                label_string = f"cnaps.io/{label_name}={label_value}"
```

If you want you can omit the 3 in split. This will cause a destructuring error if there's more than 3 underscores (you'd need to catch the exception)